spec/factories/users.rb

```diff
@@ -4,6 +4,6 @@
     name     { Faker::Name.name }
     password '12345678'
     password_confirmation '12345678'
-    admin true
+    role 1
```

Comment:
Sorry if this comes across as "arsey". I shouldn't write comments in the morning :)

This is a great piece of work, just this one thing change would make it more great IMO.